### PR TITLE
Update UI Screenshots

### PR DESCRIPTION
**Pull request by NicsunXnus**
- Reademe
- Index
- UserGuide
   - Quick Start
   - Tutorial 1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 _Reache_ is a desktop app that helps **busy working professionals** manage their large list of contacts by providing an easy-to-use interface to store contacts and organize meetings. _Reache_ is especially suited for professionals who would like to optimize their contact management by using a **Command Line Interface (CLI)**.
 
-![Ui](docs/images/Ui.png)
+![Ui](docs/images/NewUI.png)
```
Comment (ckcherry23) [inline README.md:6]: I think the name of this file should be `Ui.png` so that it can be detected by the scripts for the module website.

Comment (NicsunXnus) [reply README.md:6]: Alright will change it

File: docs/UserGuide.md [hunk 2]
```diff
@@ -97,23 +97,23 @@ Experienced users should skip this section.
...
 7. Let’s add your emails to it. Type `edit e/<YOUR PERSONAL EMAIL> l/Personal e/<YOUR WORK EMAIL> l/Work`. The updated contact details should look like this:
-![Email Added](images/Tutorial/email_edited_screen.png)
+![Email Added](images/Tutorial/Tutorial_1/new_emails_edited_screen.png)
 8. Let’s add your number to it. Type `edit ph/<YOUR MOBILE NUMBER> l/Mobile`. <br><br>
 9. Finally tag yourself using `edit t/me`. Your final contact details window should look like this:
-![Full Contact Details](images/Tutorial/tag_added.png)
+![Full Contact Details](images/Tutorial/Tutorial_1/new_tag_added_me.png)
 10. Now type `back` in the command box to return to the main window. Your **Main Window** should look like this:
-![Main Window](images/Tutorial/main_window.png)
+![Main Window](images/Tutorial/Tutorial_1/new_main_window.png)
```
Comment (ckcherry23) [inline docs/UserGuide.md:116]: Is there an issue with the `clear` command? Why is the sample meeting not cleared? :/

**Review by ckcherry23 — APPROVED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (NicsunXnus) [reply docs/UserGuide.md:116]: Nice catch, I used delete to remove the sample contacts so the meeting is still there

**Pull request merged**